src/App.jsx

```diff
@@ -14,33 +15,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortOrder, setSortOrder] = useState('none');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const getSortedGoods = (goods, order, reversed) => {
+    const sortedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (order === 'alphabetical') {
+      sortedGoods.sort();
+    } else if (order === 'length') {
+      sortedGoods.sort((a, b) => a.length - b.length);
+    }
```

Comment:
better to use switch/case here instead of multiple if/else

Reply:
Understood